### PR TITLE
feat(errors): new request errors page for mobile monitoring

### PR DESCRIPTION
**Pull request by ghost**
Jira: NR-311590

This PR includes the following changes:
- Add a new page for group request errors
- Remove the old http errors apge

### CODE REVIEW

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+## Recommended workflow [#suggested-workflow]
+
+We recommend the following approach to investigate HTTP request errors and network failures, allowing you to gain a complete understanding of the error, identify contributing factors, and implement effective debugging strategies:
+
+1. **Drill down into a single error**: To view details about a request error or network failure, click on the row from the error group table. Here you can view the request information, request attributes, and response body as well as get more information about that error.
+
+2. **Query and share error data**: To explore the data behind any of the charts or lists on the HTTP errors page:
+
+        * On any chart, click **… > <DNT>View query</DNT>**.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:40]:
```suggestion
        * On any chart, click the **…** menu, and then click **<DNT>View query</DNT>**. 
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+5. **See which filters you applied or remove filters**: The filters you select display in the filter bar. To clear filters, select the **X** next to the filter you want to clear.
+
+6. **Change the time window**: Select a new time period from the time picker dropdown.
+
+7. **View information for one specific app version**: Select the version that you want to see charts and lists for in the versions dropdown. By default, all available versions are shown.
+
+
+The sections below describe what you can do on the request errors page.
+
+## Triage errors tab [#triage-errors]
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:61]:
```suggestion
## Triage tab [#triage-errors]
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+The triage tab shows an overview of unresolved errors and how they correlate with the error rate. The error group table lists groups of errors by occurrence and allows you to asssign them to users who can investigate and fix them. 
+
+
+## Group errors tab [#group-errors]
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:66]:
```suggestion
## Group request errors tab [#group-errors]
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT>**: Select a request error to open the request errors details page.
+</figcaption>
+
+### Profiles [#error-profiles]
+
+Error profiles provide visual details about significant differences in the frequency of different values for HTTP error events. For each attribute, the error profile includes:
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:102]: @ally-sassman where is this in the UI?

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+    title="Request errors profiles"
+    style={{align: "left" }}
+    width="50%"
+    alt="Screenshot of the request errors profiles section"
+    src="/images/errors-inbox_screenshot-crop_request-errors-profiles.webp"
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT>**: Select a request error to open the request errors details page and view the heatmap distribution of the attributes.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:116]: @ally-sassman this caption doesn't quite get me to the profiles section, does it?

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+</figcaption>
+
+### Triage [#triage-errors]
+
+The triage section associates the specific error occurrence you are viewing with its [system-created error group](/docs/errors-inbox/errors-inbox/#groups). These system-created error groups are identified by a unique fingerprint. It is this unique fingerprint that allows you to triage error groups by status updates or assignments.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:121]: @ally-sassman where is this in the UI?

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+
+<img
+    title="Request errors triage page"
+    alt="Screenshot of the request errors triage page"
+    src="/images/errors-inbox_screenshot-crop_request-errors-triage-page.webp"
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a request error)**: From the request errors details page you can triage specific error occurrences by adding status updates or assignments.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:133]: @ally-sassman does this path take me to the Triage page?

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+</figcaption>
+
+### Distributed trace [#distributed-trace]
+
+The distributed trace section tracks and observes your requests as they flow through your application, traversing various services to reach completion. By visualizing the entire request path across different services, you can rapidly identify failures or performance bottlenecks.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:138]: @ally-sassman where is this in the UI? Is everything on the main page?

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+
+<img
+    title="Request errors distribute trace section"
+    alt="Screenshot of the request errors distribute trace section"
+    src="/images/errors-inbox_screenshot-crop_request-errors-distributed-trace.webp"
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a request error)**: From the request errors details page you view the distributed trace associated with that request.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:150]: @ally-sassman are distributed traces embedded in the main page?

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+</figcaption>
+
+### Response [#response]
+
+A typical response to a request comprises a response header and response body, which together convey information about the success or failure of the request. The response header contains metadata about the server, while the response body holds information about the output, including a success or failure code and corresponding message.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:155]: @ally-sassman how are "Responses" different from distributed traces?

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a request error)**: From the request errors details page you view the response body associated with that request.
+</figcaption>
+
+### Event trail [#event-trail]
+ 
+The event trail provides a chronological log of all events leading up to the request error, which helps with root cause analysis. These can be events New Relic monitors by default, or custom events. The event trail is sorted chronologically, beginning with the oldest event, which is typically the app launch, but you can modify the following:
+
+* Sort: Toggle between ascending and descending order
+* Event filtering: Filter by event type, like app launch, request, request error,  or user actions
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:175]:
```suggestion
* Event filtering: Filter by event type, like `app launch`, `request`, `request error`,  or `user actions`.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+* Event details: Expand individual events to inspect their attributes, like errorType, responseTime, or requestUrl for request events.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:176]:
```suggestion
* Event details: Expand individual events to inspect their attributes, like `errorType`, `responseTime`, or `requestUrl` for request events.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+
+After you've sorted and filtered your events, you can dig a little deeper into the events that lead up to the error by examining:
+
+* **Custom breadcrumbs**: Utilize the [Record breadcrumb SDK](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/) to create custom MobileBreadcrumb events. This allows you to log specific application interactions that may be relevant to investigating your error.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:180]:
```suggestion
* **Custom breadcrumbs**: Utilize the [Record breadcrumb SDK](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/) to create custom `MobileBreadcrumb` events. This allows you to log specific application interactions that may be relevant to investigating your error.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 14]
```diff
@@ -0,0 +1,222 @@
...
+* **Handled exceptions**: Use the [Record Handled Exception SDK](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-handled-exceptions/) methods for iOS and Android to annotate where exceptions are handled in your application. These annotations will automatically populate the crash event trail.
+
+For guidance on enhancing crash event trails with custom data, see [Record breadcrumbs](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/).
+
+To fully leverage our crash analysis tools, make sure to:
+1. Use the mobile SDK to create custom MobileBreadcrumb or MobileHandledException events.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:186]:
```suggestion
1. Use the mobile SDK to create custom `MobileBreadcrumb` or `MobileHandledException` events.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 15]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a request error)**: From the request errors details page you can dig deeper into the events that led up to a specific error using our event trail.
+</figcaption>
+
+### Attributes [#attributes]
+
+Each request error sample provides detailed information about the request, response as well as the parameters that led to the request erroring.
```
Comment (rhetoric101) [inline src/content/docs/errors-inbox/group-request-errors.mdx:202]: @ally-sassman there's no mention of attributes here--only in the figcaption.

**Review by rhetoric101 — COMMENTED** [review]
@ally-sassman thanks for adding this new doc! I inserted a few questions and suggestions.

File: src/content/docs/errors-inbox/group-request-errors.mdx (hunk 6 above)
Comment (ally-sassman) [reply src/content/docs/errors-inbox/group-request-errors.mdx:121]: These are all subsections of the request errors details page, so you'd click into an error or error group and see all of these sections on the popover (triage, profiles, heatmap, etc.)

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 16]
```diff
@@ -0,0 +1,222 @@
...
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:202]:
```suggestion
Each request error sample includes a comprehensive set of attributes that provide detailed information about the request, response, and the specific parameters that triggered the error. These attributes offer valuable insights into the error's context and help in understanding the root cause.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 17]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a request error)**: From the request errors details page you can triage specific error occurrences by adding status updates or assignments.
+</figcaption>
+
+### Distributed trace [#distributed-trace]
+
+The distributed trace section tracks and observes your requests as they flow through your application, traversing various services to reach completion. By visualizing the entire request path across different services, you can rapidly identify failures or performance bottlenecks.
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:138]:
```suggestion
When looking at a specific error, the distributed trace section tracks and observes your requests as they flow through your application, traversing various services to reach completion. By visualizing the entire request path across different services, you can rapidly identify failures or performance bottlenecks.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 18]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT>**: Select a request error to open the request errors details page and view the heatmap distribution of the attributes.
+</figcaption>
+
+### Triage [#triage-errors]
+
+The triage section associates the specific error occurrence you are viewing with its [system-created error group](/docs/errors-inbox/errors-inbox/#groups). These system-created error groups are identified by a unique fingerprint. It is this unique fingerprint that allows you to triage error groups by status updates or assignments.
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:121]:
```suggestion
When looking at a specific error, the triage section associates the specific error occurrence you are viewing with its [system-created error group](/docs/errors-inbox/errors-inbox/#groups). These system-created error groups are identified by a unique fingerprint. It is this unique fingerprint that allows you to triage error groups by status updates or assignments.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 19]
```diff
@@ -0,0 +1,222 @@
...
+    title="Request errors profiles"
+    style={{align: "left" }}
+    width="50%"
+    alt="Screenshot of the request errors profiles section"
+    src="/images/errors-inbox_screenshot-crop_request-errors-profiles.webp"
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT>**: Select a request error to open the request errors details page and view the heatmap distribution of the attributes.
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:116]:
```suggestion
    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a specific error)**: Select a request error to open the request errors details page and view the heatmap distribution of the attributes.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 20]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT>**: Select a request error to open the request errors details page.
+</figcaption>
+
+### Profiles [#error-profiles]
+
+Error profiles provide visual details about significant differences in the frequency of different values for HTTP error events. For each attribute, the error profile includes:
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:102]:
```suggestion
When looking at a specific error, the profiles section provides visual details about significant differences in the frequency of different values for HTTP error events. For each attribute, the error profile includes:
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 21]
```diff
@@ -0,0 +1,222 @@
...
+* **Update error status**: Mark the request error as **Resolved** or assign it to a specific team member who can own and examine the issue, and then deploy a fix.
+
+## Request error details [#error-details]
+
+On the request errors page, click into a specific error to see:
+* **User journeys**: Displays the different paths and actions a user performed that led to the error.
+* **All occurrences chart**: Displays the frequency of a request error over the selected period, starting from its initial detection. You can view either the aggregated data of all occurrences or a breakdown by app version.
+* **Error type breakdown**: Displays the distribution of this request errors across different operating system versions or affected devices.
+
+Resolved errors include a banner containing details on the user who resolved the request error and the resolution timestamp. Note that mobile monitoring's data retention policies apply, allowing you to filter by resolved errors for historical analysis when needed.
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:87]:
```suggestion
Resolved errors include a banner containing details on the user who resolved the request error and the resolution timestamp. Note that [mobile monitoring data retention policies](https://docs.newrelic.com/docs/data-apis/manage-data/manage-data-retention/) apply, allowing you to filter by resolved errors for historical analysis when needed.
```

**Review by rhetoric101 — DISMISSED** [review]
@ally-sassman This looks good. I just had one more small suggestion you can take or leave.

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 22]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT>**: Select a request error to open the request errors details page.
+</figcaption>
+
+### Profiles [#error-profiles]
+
+When looking at a specific error, the profiles section provides visual details about significant differences in the frequency of different values for HTTP error events. For each attribute, the error profile includes:
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:102]:
```suggestion
When you look at a specific error,, the profiles section provides visual details about significant differences in the frequency of different values for HTTP error events. For each attribute, the error profile includes:
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 23]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a specific error)**: Select a request error to open the request errors details page and view the error profiles.
+</figcaption>
+
+### Triage [#triage-errors]
+
+When looking at a specific error, the triage section associates the specific error occurrence you are viewing with its [system-created error group](/docs/errors-inbox/errors-inbox/#groups). These system-created error groups are identified by a unique fingerprint. It is this unique fingerprint that allows you to triage error groups by status updates or assignments.
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:121]:
```suggestion
When you look at a specific error, the triage section associates the specific error occurrence you are viewing with its [system-created error group](/docs/errors-inbox/errors-inbox/#groups). These system-created error groups are identified by a unique fingerprint. It is this unique fingerprint that allows you to triage error groups by status updates or assignments.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 24]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a request error)**: From the request errors details page you can triage specific error occurrences by adding status updates or assignments.
+</figcaption>
+
+### Distributed trace [#distributed-trace]
+
+When looking at a specific error, the distributed trace section tracks and observes your requests as they flow through your application, traversing various services to reach completion. By visualizing the entire request path across different services, you can rapidly identify failures or performance bottlenecks.
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:138]:
```suggestion
When you look at a specific error, the distributed trace section tracks and observes your requests as they flow through your application, traversing various services to reach completion. By visualizing the entire request path across different services, you can rapidly identify failures or performance bottlenecks.
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 25]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a request error)**: From the request errors details page you view the response body associated with that request.
+</figcaption>
+
+### Event trail [#event-trail]
+ 
+The event trail provides a chronological log of all events leading up to the request error, which helps with root cause analysis. These can be events New Relic monitors by default, or custom events. The event trail is sorted chronologically, beginning with the oldest event, which is typically the app launch, but you can modify the following:
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:172]:
```suggestion
When you look at a specific error, the event trail provides a chronological log of all events leading up to the request error, which helps with root cause analysis. These can be events New Relic monitors by default, or custom events. The event trail is sorted chronologically, beginning with the oldest event, which is typically the app launch, but you can modify the following:
```

File: src/content/docs/errors-inbox/group-request-errors.mdx [hunk 26]
```diff
@@ -0,0 +1,222 @@
...
+/>
+
+
+<figcaption>
+    **<DNT>[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > Request errors</DNT> > (Select a request error)**: From the request errors details page you can dig deeper into the events that led up to a specific error using our event trail.
+</figcaption>
+
+### Attributes [#attributes]
+
+Each request error sample includes a comprehensive set of attributes that provide detailed information about the request, response, and the specific parameters that triggered the error. These attributes offer valuable insights into the error's context and help in understanding the root cause.
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/group-request-errors.mdx:202]:
```suggestion
When you look at a specific error, each request error sample includes a comprehensive set of attributes that provide detailed information about the request, response, and the specific parameters that triggered the error. These attributes offer valuable insights into the error's context and help in understanding the root cause.
```

**Pull request merged**